app/models/import/row.rb

```diff
@@ -3,6 +3,7 @@ class Import::Row < ApplicationRecord
 
   validates :amount, numericality: true, allow_blank: true
   validates :currency, presence: true
+  validates :exchange_operating_mic, presence: true, if: -> { import.type == "TradeImport" && import.exchange_operating_mic_col_label.present? }
```

Comment:
Let's remove this so that users can import a CSV of trades where _some_ have exchange mic codes provided (will have historical prices) while others don't (custom, or unknown tickers).

Our backend logic during our "sync" process will handle this okay.